src/components/Page.js

```diff
@@ -1,22 +1,52 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 import Sidebar from './Sidebar';
+import MobileMenuBar from './MobileMenuBar';
+import { mediaQueries } from './Common';
 
 const Container = styled.div`
   display: flex;
   align-items: stretch;
   min-height: 100vh;
 `;
 
+const LeftColumn = styled.div`
+  box-sizing: border-box;
+  width: 100%;
+`;
+
 const Content = styled.div`
   padding: 24px 60px;
   box-sizing: border-box;
   width: 100%;
 `;
 
-export default ({ children }) => (
-  <Container>
-    <Sidebar />
-    <Content>{children}</Content>
-  </Container>
-);
\ No newline at end of file
+
+
+const Page = ({ children }) => {
+  const SidebarContainer = styled.div`
```

Comment:
best not to declare this inside the Page component

Reply:
i moved it back to Sidebar.js (forgot that i could just pass in the show/hide prop instead of pulling the container up to the Page.js file)